lib/index.js

```diff
@@ -57,6 +58,38 @@ function setupTracker(spec) {
   };
 }
 
+function checkDataFiles() {
+  const { ECOBALYSE_DATA_DIR } = process.env;
+
+  if (!ECOBALYSE_DATA_DIR) {
+    console.error(
+      `
+🚨 ERROR: You need to specify ECOBALYSE_DATA_DIR env variable.
+   It must point to a directory with the detailed versions of the processes files. Please, edit your .env file accordingly.",
+   -> Exiting the process.
+`,
+    );
+    process.exit(1);
+  }
```

Comment:
In the same spirit, how about throwing instead of printing and exiting here?

In the same spirit, how about throwing instead of printing and exiting here?
```js
if (!ECOBALYSE_DATA_DIR) {
  throw new Error("You need to specify…");
}
```

So later you can catch and exit:

```js
try {
  checkDataFiles(process.env);
} catch (err) {
  console.error(`🚨 ERROR: ${err.message}`);
  process.exit(1);
}
```